app/assets/stylesheets/partials/authoring/modal_edit.scss

```diff
@@ -8,9 +8,10 @@
   position: fixed !important;
   top: 20px !important;
 
-  form[class^="edit_"] {
+  form[class^="edit_"], form.styled-admin-form {
```

Comment:
This is the fix for the problem with library interactive forms. The other changes are all for the export, import, and publish modal layout clean up.